lib/seatch_repository_screen.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:count_up_app/get_issues.dart';
+import 'package:flutter/material.dart';
+
+class SearchRepositoryScreen extends StatefulWidget {
+  const SearchRepositoryScreen({super.key});
+
+  @override
+  State<SearchRepositoryScreen> createState() => _SearchRepositoryScreenState();
+}
+
+class _SearchRepositoryScreenState extends State<SearchRepositoryScreen> {
+  final ownerController = TextEditingController();
+  final repositoryController = TextEditingController();
```

Comment:
TextEditingControllerは必ずdisposeしてください。
```dart
  @override
  void dispose() {
    _controller.dispose();
    super.dispose();
  }
```

> Remember to [dispose](https://api.flutter.dev/flutter/foundation/ChangeNotifier/dispose.html) of the [TextEditingController](https://api.flutter.dev/flutter/widgets/TextEditingController-class.html) when it is no longer needed. This will ensure we discard any resources used by the object.

https://api.flutter.dev/flutter/widgets/TextEditingController-class.html